SmilesMiles/View/NewTrip.swift

```diff
@@ -10,17 +10,18 @@ import SwiftUI
 struct NewTrip: View {
     @Binding var trips: [String]
```

Comment:
Check if trips can be accessed with an instance of NewTripViewModel.

Reply:
It can be,  however, this approach creates a tight coupling between the two view models and can make it harder to reason about the state of the application.

Reply:
Modified